docs/root/start/simple_example.rst

```diff
@@ -0,0 +1,240 @@
+  consider for this listener.
+  :ref:`FilterChain <envoy_api_msg_listener.FilterChain>` with the most specific
+  :ref:`FilterChainMatch <envoy_api_msg_listener.FilterChainMatch>` criteria is used on a
+  connection. This simple example does not use explicit match criteria.
+
+.. code-block:: yaml
+
+        - filters:
+          - name: envoy.http_connection_manager
+            config:
+
+..
+
+  The filter chain consists of a list of ``filters``.
+  Filters can be :ref:`network (L3/L4) filters <arch_overview_network_filters>`, the
+  :ref:`HTTP Connection Manager <arch_overview_http_conn_man>` (which is a L4 filter),
+  or :ref:`HTTP filters <arch_overview_http_filters>`.
+
+  The ``name`` specifies a supported filter.
+  Envoy has several built in filters that start with `envoy`. The specified filter is the
+  :ref:`HTTP Connection Manager <config_http_conn_man>` which does all manner of things to manage HTTP
+  connections.
+
+  The ``config`` contains the filter-specific configuration.
+
+.. code-block:: yaml
+
+              stat_prefix: ingress_http
+              route_config:
+                name: local_route
+                virtual_hosts:
+                - name: local_service
+                  domains: ["*"]
+                  routes:
+                  - match: { prefix: "/" }
+                    route: { host_rewrite: www.google.com, cluster: service_google }
+              http_filters:
+              - name: envoy.router
+
+..
+
+  This is the configuration for the :ref:`HTTP Connection Manager <config_http_conn_man>`.
+
+  The ``stat_prefix`` field is a human readable prefix used to identify this connection manager in
+  the :ref:`statistics <config_http_conn_man_stats>`.
+
+  The ``route_config`` field specifies a static :ref:`Route configuration <envoy_api_file_envoy/api/v2/rds.proto>`.
+
+  The ``name`` field names the route configuration. This is optional.
+
+  The ``virtual_hosts`` field contains a list of :ref:`Virtual Hosts <envoy_api_msg_route.VirtualHost>`.
+
+  The ``name`` field names the virtual host for statistics reporting.
+
+  The ``domains`` field contains a list of domains (host/authority header) that will be matched to this virtual
+  host.
+
+  The ``routes`` field contains a list of :ref:`Routes <envoy_api_msg_route.Route>` that are matched, in order, for incoming requests. The first
+  route that matches is used.
+
+  The ``match`` field provides the :ref:`RouteMatch parameters <envoy_api_msg_route.RouteMatch>` used to select the route.
+
+  The ``route`` field provides the :ref:`RouteAction parameters <envoy_api_msg_route.RouteAction>` invoked when the route is selected.
+
+  The ``http_filters`` field contains a list of
+  :ref:`HttpFilter <envoy_api_msg_config.filter.network.http_connection_manager.v2.HttpFilter>` for the connection manager.
+  In this case
+  the built-in :ref:`envoy.router <config_http_filters_router>` filter is specified which
```

Comment:
By 'built-in' do you mean one included in the default binary?

Reply:
Yes, that's the way the other documentation speaks about it I think.